anime-list-master.xml

```diff
@@ -4916,7 +4916,7 @@
   <anime anidbid="1351" tvdbid="73518" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Mitsubachi Maaya no Bouken</name>
   </anime>
-  <anime anidbid="1352" tvdbid="unknown" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="1352" tvdbid="250580" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>8 Man</name>
   </anime>
   <anime anidbid="1353" tvdbid="250580" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
```

Comment:
Same TVDB ID and Season. That's a conflict

Reply:
Corrected with https://thetvdb.com/series/8-man-after (394945)

Reply:
Seems the same episodes are also in the first ID's s00e02-05, the old dilemma that has no right answer.
Either option is valid.

Reply:
Yeah, I'm thinking with of Default to the new show (if it exists) for better poster/image support that would come with them being separate shows. But both should resolve.